File: app/src/main/java/com/section/recyclerview/ActivityModel.java
/*
 * Copyright 2021 ~ https://github.com/braver-tool ~All rights reserved
 */

package com.section.recyclerview;

public class ActivityModel {

    private String ActivityName;
    private String Calories;
    private String Duration;
    private String ActivityTime;
    private String headerDate;

    public ActivityModel() {
    }

    public ActivityModel(String activityName, String calories, String duration, String activityTime) {
        ActivityName = activityName;
        Calories = calories;
        Duration = duration;
        ActivityTime = activityTime;
        headerDate = "";
    }

    public String getHeaderDate() {
        return headerDate;
    }

    public void setHeaderDate(String headerDate) {
        this.headerDate = headerDate;
    }

    public String getActivityName() {
        return ActivityName;
    }

    public String getCalories() {
        return Calories;
    }

    public String getDuration() {
        return Duration;
    }

    public String getActivityTime() {
        return ActivityTime;
    }

}

File: appKotlin/src/main/java/com/section/recyclerview/kotlin/ActivityModel.kt
/*
 * Copyright 2021 ~ https://github.com/braver-tool ~All rights reserved
 */

package com.section.recyclerview.kotlin

class ActivityModel {
    private var ActivityName: String? = null
    private var Calories: String? = null
    private var Duration: String? = null
    private var ActivityTime: String? = null
    private var headerDate: String? = null

    constructor() {}
    constructor(activityName: String?, calories: String?, duration: String?, activityTime: String?) {
        ActivityName = activityName
        Calories = calories
        Duration = duration
        ActivityTime = activityTime
        headerDate = ""
    }

    fun getHeaderDate(): String? {
        return headerDate
    }

    fun setHeaderDate(headerDate: String?) {
        this.headerDate = headerDate
    }

    fun getActivityName(): String? {
        return ActivityName
    }

    fun getCalories(): String? {
        return Calories
    }

    fun getDuration(): String? {
        return Duration
    }

    fun getActivityTime(): String? {
        return ActivityTime
    }
}